squarelet/organizations/serializers.py

```diff
@@ -177,6 +177,7 @@ class PressPassNestedInvitationSerializer(serializers.ModelSerializer):
     class Meta:
         model = Invitation
         fields = (
+            "uuid",
```

Comment:
Why was UUID added here?

Reply:
Ah thanks for catching this change - I just removed it. I was trying to understand how the invitations worked for building the PP front-end and thought I needed the unique ID... but I do not.